MSAL/test/integration/native_auth/common/Model.swift

```diff
@@ -45,16 +45,16 @@ enum MockAPIEndpoint: String {
 }
 
 enum MockAPIResponse: String {
+    case signUpStartSuccess = "SignUpStartSuccess"
     case invalidRequest = "InvalidRequest"
     case invalidToken = "InvalidToken"
     case invalidClient = "InvalidClient"
     case invalidGrant = "InvalidGrant"
     case invalidScope = "InvalidScope"
     case expiredToken = "ExpiredToken"
-    case unsupportedAuthMethod = "UnsupportedAuthMethod"
+    case unsupportedAuthMethod = "AuthNotSupported"
```

Comment:
should we change the name of this case to "authNotSupported"?

Reply:
Done.